scripts/remove-idle-users.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import argparse
+import sys
+import json
+import urllib
+import requests
+import re
+
+## debug request
+import httplib as http_client
+http_client.HTTPConnection.debuglevel = 1
+
+def get_room_id(homeserver, alias, token):
+    res = requests.get(homeserver + "/_matrix/client/r0/directory/room/" + urllib.quote(alias) + "?access_token=" + token).json()
+    return res.get("room_id", None)
```

Comment:
Ah but then you don't get to see the error. Fair point, I'll have it raise for status.